### DISCUSSION TITLE
open source

### DISCUSSION THREAD

**Discussion by Xionse** [Software-Savants/.github#2] [Q&A] [answered]
Why should open source really be promoted?

**Comment by AnthonyByansi** [comment, marked as answer]
Cost: Open-source software is usually free to use, modify, and distribute, which can be a significant cost advantage for individuals and organizations.

Innovation: Open-source software encourages collaboration and the sharing of ideas, which can lead to the development of better software.

Security: Because open-source software is developed and reviewed by a large community of developers, it can be more reliable and secure than proprietary software.

Accessibility: Open-source software can be a powerful tool for promoting digital literacy and inclusivity, by providing access to software and technology to a wider range of people.

Freedom: Using open-source software gives users the freedom to modify and customize the software to meet their specific needs and goals

**Reply by AnthonyByansi** [reply]
hopefully, value is being attached. Can't wait for you guys to be part of this eco system

**Reply by Xionse** [reply]
All has made sense. Thanks
